descriptors/ModuleDescriptor-template.json

```diff
@@ -236,6 +236,12 @@
           "pathPattern" : "/users/expire/timer",
           "unit" : "minute",
           "delay" : "1"
+        },
+        {
+          "methods" : [ "POST" ],
+          "pathPattern" : "/users/users-outbox/process",
```

Comment:
What is the significance of the second `users` in this path?

Is there intended to be a separate outbox for each record type in the future?